mxcube3/core/scutils.py

```diff
@@ -19,6 +19,7 @@ def init_signals():
 
     """Initialize hwobj signals."""
     blcontrol.sample_changer.connect('stateChanged', signals.sc_state_changed)
+    blcontrol.sample_changer.connect('pathSafeChanged', signals.path_safe_changed)
```

Comment:
Do you think we can come up with a name that is more general name
than pathSafeChanged maybe isSafe or safe ?

Reply:
you mean for the signal? safeAreaChanged maybe?

Reply:
Yeah, I mean it should probably even be part of generic sample changer, it would be great if both signal and handler had a more generic name. I guess its in reality implemented as a "safe volume", but we don't really care about that only that the sample changer "is safe".

Reply:
We do have the safe area, its for the moment linked to the busy state so that's how its handled today but we could definitely use a feature like this. I think its general to most sample changers using a robotic arm that they have a "safe zone". My comment is simply that we could perhaps find a more general name than pathSafeXXX, the intention is in the end to indicate that the sample changer or parts of it in some notion is "safe". Simply isSafe, isCollisionSafe or something in that direction would seem logical to me ?

Reply:
The business about the safe area in our case is mostly used to speed up the loading - in the sample changed hardware object we start actions once the sample is on the gonio head. I would agree with @marcus-oscarsson that we need something more generic like isCollisionSafe.

Reply:
changed to `isCollisionSafe `